app/controllers/api/v1/planning_applications_controller.rb

```diff
@@ -22,18 +22,6 @@ def decision_notice
         @blank_layout = true
       end
 
-      def create
-        @planning_application = PlanningApplicationCreationService.new(
-          local_authority: current_local_authority, params:, api_user: current_api_user
-        ).call
-
-        send_success_response
-
-        post_application_to_staging if Bops.env.production?
-      rescue PlanningApplicationCreationService::CreateError => e
-        send_failed_response(e, params)
-      end
-
```

Comment:
It looks like we can probably also remove the other methods called from this action: `send_success_response` and `send_failed_response`.

`send_not_found_response` will still be needed for `show`.